project/Dependencies.scala

```diff
@@ -8,7 +8,7 @@ object Dependencies {
 
   val sparkVersion = "2.1.0"
   val scalaTestVersion = "3.0.0"
-  val tensorflowVersion = "0.12.head"
+  val tensorflowVersion = "1.1.0"
```

Comment:
I'll open up another issue about this and put in the code I used for `getFields`